<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: &quot;dark blue";;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="spushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>Начать заново</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hitMeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>570</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>Взять карту</string>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Соперник</string>
    </property>
   </widget>
   <widget class="QLabel" name="plabel">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>280</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Игрок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="standButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>570</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: light gray;</string>
    </property>
    <property name="text">
     <string>Закончить</string>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
